sources/models/Cargo.toml

```
@@ -10,24 +10,12 @@ build = "build.rs"
 exclude = ["README.md"]
 
 [dependencies]
-base64 = "0.13"
 bottlerocket-release = { path = "../bottlerocket-release", version = "0.1" }
-indexmap = { version = "1", features = ["serde"] }
-lazy_static = "1"
 libc = "0.2"
 model-derive = { path = "model-derive", version = "0.1" }
-regex = "1"
-scalar = { path = "scalar", version = "0.1" }
-scalar-derive = { path = "scalar-derive", version = "0.1" }
-semver = "1"
+modeled_types = { path = "modeled_types", version = "0.1" }
```

For consistency? We tend to use the kebab form elsewhere.

```suggestion
modeled-types = { path = "modeled-types", version = "0.1" }
```